### PR TITLE
Added Docker file

### PR DESCRIPTION
**Pull request by ccheraa**
The Docker image is for the demo editor to test `prosemirror-jdita` functionalities

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,78 +1,20 @@
...
-  "name": "prosemirror-jdita",
-  "version": "0.0.1",
-  "description": "Use JDita in prosemirror",
-  "main": "./lib/index.js",
-  "types": "./lib/index.d.ts",
+  "name": "prosemirror-jdita-package",
+  "version": "0.1.1",
+  "description": "Prosemirror plugin to edit JDita documents",
+  "main": "index.js",
+  "repository": "https://github.com/ccheraa/prosemirror-jdita.git",
```
Comment (adamretter) [inline package.json:6]: This needs some fixing from the left-version. Can you please set:

```json
"repository": {
    "type": "git",
    "url": "git+https://github.com/evolvedbinary/prosemirror-jdita.git"
  },
  "keywords": [	
    "LwDITA",	
    "DITA",	
    "XML",	
    "JSON"	
  ],	
  "author": "Evolved Binary",	
  "license": "AGPL-3.0",	
  "bugs": {	
    "url": "https://github.com/evolvedbinary/prosemirror-jdita/issues"	
  },	
  "homepage": "https://github.com/evolvedbinary/prosemirror-jdita#readme"
```

Comment (ccheraa) [reply package.json:6]: Oh yes, I forgot about it.
It should be changed in the other 2 `package.json` files for the 2 workspaces

File: prosemirror-jdita-example/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FROM nginx:alpine
```
Comment (adamretter) [inline prosemirror-jdita-example/Dockerfile:1]: Dockerfile looks good, but we need a bit more in there as these are likely to be published somewhere like Dockerhub... can you also add:

```
LABEL org.label-schema.build-date=${build-tstamp} \
      org.label-schema.description="ProseMirror JDita Example" \
      org.label-schema.name="prosemirror-jdita-example" \
      org.label-schema.schema-version="1.0" \
      org.label-schema.url="https://github.com/evolvedbinary/prosemirror-jdita" \
      org.label-schema.vcs-ref=${build-commit-abbrev} \
      org.label-schema.vcs-url="https://github.com/evolvedbinary/prosemirror-jdita" \
      org.label-schema.vendor="Evolved Binary"

COPY LICENSE /usr/share/nginx/html/LICENSE
```

I am not sure how you template in the `${value}` bits with Yarn/Node.js any ideas?

File: prosemirror-jdita-example/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "prosemirror-jdita-example",
+  "version": "0.1.1",
+  "description": "Example for Prosemirror plugin",
+  "main": "index.js",
+  "repository": "https://github.com/ccheraa/prosemirror-jdita.git",
```
Comment (adamretter) [inline prosemirror-jdita-example/package.json:6]: "repository": {
    "type": "git",
    "url": "git+https://github.com/evolvedbinary/prosemirror-jdita.git"
  },
  "keywords": [	
    "ProseMirror",
    "Docker",
    "LwDITA",	
    "DITA",	
    "XML",	
    "JSON"	
  ],	
  "author": "Evolved Binary",	
  "license": "AGPL-3.0",	
  "bugs": {	
    "url": "https://github.com/evolvedbinary/prosemirror-jdita/issues"	
  },	
  "homepage": "https://github.com/evolvedbinary/prosemirror-jdita#readme"

File: prosemirror-jdita-example/Dockerfile (hunk 2 above)
Comment (ccheraa) [reply prosemirror-jdita-example/Dockerfile:1]: We can run a script that updates Dockerfile after each build

File: prosemirror-jdita/package.json [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    "lint:fix": "eslint src --ext .ts --fix",
+    "build": "rimraf ./lib && tsc",
+    "prepublishOnly": "yarn build",
+    "test": "mocha",
+    "test:watch": "mocha -w",
+    "coverage": "rimraf coverage && nyc yarn mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ccheraa/prosemirror-jdita.git"
```
Comment (adamretter) [inline prosemirror-jdita/package.json:20]: please replace `cchreaa` with `evolvedbinary`

**Review by adamretter — CHANGES_REQUESTED** [review]
Just some small tweaks please.

Also I see at least two files that were moved where they should not have been:
1. `LICENSE`
2. `.clabot`

File: prosemirror-jdita-example/docker.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
```
Comment (adamretter) [inline prosemirror-jdita-example/docker.js]: Ideally we should start from a template Dockerfile, and just template in the `${value}` bits. How hard would that be?

But... perhaps we should do it in the next PR?

**Review by adamretter — APPROVED** [review]

**Pull request merged**